kensu/numpy/__init__.py

```diff
@@ -409,17 +406,21 @@ def wrapper(*args, **kwargs):
 
             new_args = tuple(new_args)
 
-            result = method(*new_args, **kwargs)
-
-            original_result = result
-
-            return ndarray.using(result)
+            original_result = method(*new_args, **kwargs)
+            ksu_result = ndarray.using(original_result)
+            # FIXME: don't we need to call numpy_report here to attach the lineage?
```

Comment:
FIXME?! do you remember why it was omitted? @SammyEK

Reply:
I think it was because the cases we encountered were np.array() of lists, and those had no lineage, as the data were created

Reply:
trying to fix/hack it...